trySwift/ScheduleViewController.swift

```diff
@@ -77,19 +81,3 @@ private extension ScheduleViewController {
     }
 }
 
-extension ScheduleViewController: UITabBarControllerDelegate {
```

Comment:
see the comment above, this functionality is handled in a `RootTabBarController` now